legend-engine-xt-serviceStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/service/plugin/ServiceStoreExecutor.java

```diff
@@ -19,11 +19,11 @@
 
 public class ServiceStoreExecutor implements StoreExecutor
 {
-    static final ServiceStoreExecutor INSTANCE = new ServiceStoreExecutor(new ServiceStoreState());
+    static final ServiceStoreExecutor INSTANCE = new ServiceStoreExecutor(new ServiceStoreState(ServiceStoreExecutionConfiguration.newInstance().build()));
```

Comment:
I don't think this is valid any longer. Earlier we could just use this because there was no config. I believe we will now need to organize it better, wherein we create instance separatly instead of singleton